src/main/java/aeneas/models/LevelGenerator.java

```diff
@@ -14,70 +14,80 @@ public static void main(String[] args) {
     File file = new File("build/levels/default/");
     file.mkdirs();
 
-    for (Level l : generateDefaultLevels()){
+    for (Level l : generateDefaultLevels()) {
       saveNewLevel(l);
     }
   }
 
-  public static ArrayList<Level> generateDefaultLevels(){
+  public static ArrayList<Level> generateDefaultLevels() {
     ArrayList<Level> levels = new ArrayList<Level>();
 
-    levels.add(newPuzzle(16, new int[]{0,1,2,3,2}));
-    levels.add(newLightning(60, new int[]{8,6,4,11,15}));
-    levels.add(newRelease(16, new int[]{0,1,2,3,2}));
-
-    levels.add(newPuzzle(12, new int[]{4,2,12,2,2}));
-    levels.add(newLightning(50, new int[]{2,0,1,10,14}));
-    levels.add(newRelease(12, new int[]{4,2,12,2,2}));
-
-    levels.add(newPuzzle(12, new int[]{10,11,2,14,2}));
-    levels.add(newLightning(40, new int[]{1,6,4,11,13}));
-    levels.add(newRelease(12, new int[]{10,11,2,14,2}));
-
-    levels.add(newPuzzle(10, new int[]{4,13,7,4,8}));
-    levels.add(newLightning(30, new int[]{0,3,3,11,12}));
-    levels.add(newRelease(10, new int[]{4,13,7,4,8}));
-
-    levels.add(newPuzzle(10, new int[]{3,6,5,4,9}));
-    levels.add(newLightning(20, new int[]{1,4,1,11,13}));
-    levels.add(newRelease(10, new int[]{3,6,5,4,9}));
+    levels.add(newPuzzle(30, new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, }));
+    levels.add(newLightning(90, new int[] { 8, 6, 4, 11, 15 }));
+    levels.add(newRelease(16, new int[] { 0, 1, 2, 3, 2 }));
+
+    levels.add(newPuzzle(30,
+        new int[] { 4, 0, 12, 2, 30, 10, 23, 5, 14, 22, 11, 6, 7, 24, 5, 10, 12,
+            15, 1, 21, 2, 3, 4, 5, 8, 7, 28, 29, 10, 34, 12, 13, 14, 15, 12, 14,
+            27, 2, 5 }));
+    levels.add(newLightning(85, new int[] { 2, 0, 1, 10, 14 }));
+    levels.add(newRelease(12, new int[] { 4, 2, 12, 2, 2 }));
+
+    levels.add(newPuzzle(30,
+        new int[] { 10, 11, 2, 14, 22, 21, 0, 2, 6, 3, 4, 27, 5, 26, 7, 12, 8,
+            1, 9, 14, 1, 4, 0, 6, 25, 2, 0, 12, 6, 4, 12, 4, 23, 24, 12, 12, 12,
+            31, 34 }));
+    levels.add(newLightning(80, new int[] { 1, 6, 4, 11, 13 }));
+    levels.add(newRelease(12, new int[] { 10, 11, 2, 14, 2 }));
+
+    levels.add(newPuzzle(25, new int[] { 31, 4, 15, 9, 26, 5, 34, 8, 9, 7, 9,
+        32, 3, 8, 4, 6, 26, 4, 33, 2, 4, 6, 15, 16, 23, 32, 1, 2, 5, 8, 33, 25, 23, 20, 14,
+        7, 0, 0, 10, 11, 12, 13, 17, 18, 19}));
+    levels.add(newLightning(70, new int[] { 0, 3, 3, 11, 12 }));
+    levels.add(newRelease(10, new int[] { 4, 13, 7, 4, 8 }));
+
+    levels.add(newPuzzle(25,
+        new int[] { 4, 2, 12, 2, 2, 10, 3, 5, 14, 12, 11, 6, 7, 5, 5, 10, 12,
+            15, 1, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 12, 14, 3,
+            2, 5 }));
+    levels.add(newLightning(60, new int[] { 1, 4, 1, 11, 13 }));
+    levels.add(newRelease(10, new int[] { 3, 6, 5, 4, 9 }));
 
     return levels;
   }
 
   private static ReleaseLevel newRelease(int moves, int[] pieceIndeces) {
```

Comment:
Can you initialize the release with at least one number? As is, that will work, but the problem is right now you start with 3 stars and that causes the level to immediately exit.